### PR TITLE
Show correct vis. on dashboard

### LINKED ISSUE #82: Display visualisation selected by user (and not always thresholdchart) 📊
### Description
Make chart type dependent on vis. selected -- i.e. information available in global store after #85.

#### Conciderations

It might be easier to have all visualisations accept an object (i.e. _one_ argument) and use some clever mapping between input and output.

### Subtasks
- [ ] Decide what type of visualisation should be shown
- [ ] Map from raw data to visualisation input

### LINKED ISSUE #85: Add the selected visualisation type to the global store
### Description
The user should be able to select a visualisation from the ones generated in #84, and this information should be passed to the global store.

### Subtasks
- [ ] Allow user to select visualisation
- [ ] Add information to global data store model

### LINKED ISSUE #86: Use custom ID instead of title for determining data source
### Description
Right now data source is based on the title of the metadata entry that is returned[0]. This is error-prone as some of the titles are quite long and complicated.

It would be better to migrate to a more uniform ID format.

[0] https://github.com/IT2901-SINTEF01/frontend/pull/81

### Subtasks
- [ ] Review code touched in aforementioned PR to use ID instead of title

### Related sources
- Blocked by https://github.com/IT2901-SINTEF01/backend/issues/93

### PR DESCRIPTION
**Pull request by FredrikAugust**
Closes #85 
Closes #82

Changelog
-----------

- Use the new datasource ID instead of name for mapping keys
- Default to get population for one municipality and a handful of years and map to ChartEntry[]
- Store visualisation type in store
- Show correct visualisation on dashboard

### CODE REVIEW

File: src/components/organisms/Dashboard.tsx [hunk 1]
```diff
@@ -68,13 +70,28 @@ const Dashboard: React.FC = () => {
...
+                                                    case VisualisationType.THRESHOLD:
+                                                        return (
+                                                            <ThresholdChart
+                                                                data={data}
+                                                                height={parent.height}
+                                                                width={parent.width}
+                                                            />
+                                                        );
+                                                    default:
+                                                        return <Text>Visualisation currently not supported.</Text>;
```
Comment (blauks) [inline src/components/organisms/Dashboard.tsx]: Translate to norwegian :)

**Review by blauks — COMMENTED** [review]
Looks very LGTM to me!
_But,_ we should probably keep all error/feedback messages in the same language

File: src/components/organisms/Dashboard.tsx (hunk 1 above)
Comment (FredrikAugust) [reply src/components/organisms/Dashboard.tsx]: Yep. Please request changes when you encounter an error.

Comment (FredrikAugust) [reply src/components/organisms/Dashboard.tsx]: I.e. don't "comment".

**Review by blauks — APPROVED** [review]
Very nice __Borat voice__

**Pull request merged**